tests/EdFi.Ods.WebApi.IntegrationTests/Controllers/Standard/5.1.0/AggregateDependencyControllerTests.Should_Get_Dependencies.approved.txt

```diff
@@ -175,6 +175,14 @@
       "Update"
     ]
   },
+  {
```

Comment:
This "approved" file needed to be updated to reflect changes from DS 5.0.0 -> 5.1.0. It had not been in use since the change to DS 5.1.0 because the containing folder was named 5.0.0.